plugins/database/mssql/mssql_test.go

```diff
@@ -411,6 +458,28 @@ func assertCredsDoNotExist(t testing.TB, connURL, username, password string) {
 	}
 }
 
+func assertContainedDBCredsDoNotExist(t testing.TB, connURL, username string) {
+	t.Helper()
+	err := testContainedDBCredsExist(connURL, username)
+	assert.EqualError(t, err, "mssql: Cannot drop the user 'vaultuser', because it does not exist or you do not have permission.")
+}
+
+func testContainedDBCredsExist(connURL, username string) error {
+	ctx := context.Background()
+	// Log in
+	db, err := sql.Open("mssql", connURL)
+	if err != nil {
+		return err
+	}
```

Comment:
Don't forget to close the connection:

```suggestion
	db, err := sql.Open("mssql", connURL)
	if err != nil {
		return err
	}
	defer db.Close()
```

Reply:
Updated in cf19eef